### PR TITLE
Season stats new

### PR DESCRIPTION
**Pull request by katemorris**
All of the season statistics just in the manager. Will need to move them to the stat tracker in next step and return name. 

winningest_coach
worst_coach
most_accurate_team
least_accurate_team
most_tackles
fewest_tackles

### CODE REVIEW

File: test/team_manager_test.rb [hunk 1]
```diff
@@ -43,7 +43,7 @@ def test_find_worst_offense
   end
 
   def test_average_number_of_goals_scored_by_team_by_type
-    assert_equal 2.00, @team_manager.average_number_of_goals_scored_by_team_by_type('3', 'away')
+    assert_equal 1.67, @team_manager.average_number_of_goals_scored_by_team_by_type('3', 'away')
```
[the remark this answers is not in the document]
Comment (cowens87) [reply test/team_manager_test.rb:46]: We may look at method naming conventions here to cut down on characters

File: test/stat_tracker_test.rb [hunk 2]
```diff
@@ -35,11 +35,11 @@ def test_it_can_find_a_name
   end
 
   def test_average_number_of_goals_scored_by_team
-    assert_equal 1.6, @stat_tracker.average_number_of_goals_scored_by_team('3')
+    assert_equal 1.5, @stat_tracker.average_number_of_goals_scored_by_team('3')
   end
 
   def test_average_number_of_goals_scored_by_team_by_type
-    assert_equal 2.00, @stat_tracker.average_number_of_goals_scored_by_team_by_type('3', 'away')
+    assert_equal 1.67, @stat_tracker.average_number_of_goals_scored_by_team_by_type('3', 'away')
```
[the remark this answers is not in the document]
Comment (cowens87) [reply test/stat_tracker_test.rb:42]: We may look at method naming conventions here to cut down on characters

File: test/game_teams_manager_test.rb [hunk 3]
```diff
@@ -8,40 +8,156 @@
...
+  def test_tackles_by_team
+    season_id = '20122013'
+    team_num = '3'
+    assert_equal 179, @game_teams_manager.tackles_by_team(season_id, team_num)
+  end
+
+  def test_teams_hash_w_tackles
+    season_id = '20122013'
+    expected = {
+                  '3' => 179,
```
[the remark this answers is not in the document]
Comment (cowens87) [reply test/game_teams_manager_test.rb:146]: Easy fix, just bring '3' in on indentation

File: test/game_teams_manager_test.rb [hunk 4]
```diff
@@ -8,40 +8,156 @@
...
+  end
+
+  def test_teams_hash_w_ratio_shots_goals
+    season_id = '20122013'
+    expected = {
+                  '3' => 0.21,
+                  '6' => 0.33,
+                  '5' => 0.08
+    }
+    assert_equal expected, @game_teams_manager.teams_hash_w_ratio_shots_goals(season_id)
```
[the remark this answers is not in the document]
Comment (cowens87) [reply test/game_teams_manager_test.rb:124]: We may look at method naming conventions here to cut down on characters

[the remark this answers is not in the document]
Comment (cowens87) [reply test/game_teams_manager_test.rb:124]: Easy fix, just bring '3' in on indentation

File: test/game_teams_manager_test.rb [hunk 5]
```diff
@@ -8,40 +8,156 @@
...
+  def test_shot_goal_ratio
+    season_id = '20122013'
+    team_num = '3'
+    assert_equal 0.21, @game_teams_manager.shot_goal_ratio(season_id, team_num)
+  end
+
+  def test_teams_hash_w_ratio_shots_goals
+    season_id = '20122013'
+    expected = {
+                  '3' => 0.21,
```
[the remark this answers is not in the document]
Comment (cowens87) [reply test/game_teams_manager_test.rb:120]: Easy fix, just bring '3' in on indentation

File: test/game_teams_manager_test.rb [hunk 6]
```diff
@@ -8,40 +8,156 @@
...
-    assert_equal 4, @game_teams_manager.total_goals_by_type('3', 'away')
-    assert_equal 3, @game_teams_manager.total_goals_by_type('3', 'home')
+    assert_equal 5, @game_teams_manager.total_goals_by_type('3', 'away')
+    assert_equal 4, @game_teams_manager.total_goals_by_type('3', 'home')
   end
 
   def test_average_number_of_goals_scored_by_team_by_type
-    assert_equal 2.00, @game_teams_manager.average_number_of_goals_scored_by_team_by_type('3', 'away')
-    assert_equal 1.50, @game_teams_manager.average_number_of_goals_scored_by_team_by_type('3', 'home')
+    assert_equal 1.67, @game_teams_manager.average_number_of_goals_scored_by_team_by_type('3', 'away')
```
[the remark this answers is not in the document]
Comment (cowens87) [reply test/game_teams_manager_test.rb:51]: We may look at method naming conventions here to cut down on characters

File: test/game_teams_manager_test.rb [hunk 7]
```diff
@@ -8,40 +8,156 @@
...
   end
 
   def test_return_total_goals_by_team
-    assert_equal 15, @game_teams_manager.total_goals('6')
+    assert_equal 20, @game_teams_manager.total_goals('6')
   end
 
   def test_average_number_of_goals_scored_by_team
-    assert_equal 2.5, @game_teams_manager.average_number_of_goals_scored_by_team('6')
+    assert_equal 2.86, @game_teams_manager.average_number_of_goals_scored_by_team('6')
```
[the remark this answers is not in the document]
Comment (cowens87) [reply test/game_teams_manager_test.rb:37]: We may look at method naming conventions here to cut down on characters

File: test/game_team_test.rb [hunk 8]
```diff
@@ -0,0 +1,48 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/stat_tracker'
+require './lib/game_teams_manager'
+require './lib/game_team'
+require 'pry';
+require 'mocha/minitest'
+
+class TeamTest < Minitest::Test
+  def setup
```
[the remark this answers is not in the document]
Comment (GeorgieGirl24) [reply test/game_team_test.rb:10]: we will need to get rid of empty lines as we continue to refactor

**Pull request merged**